### DISCUSSION TITLE
Is there something similar to redux-form FormSections /> in RHF?

### DISCUSSION THREAD

**Discussion by ScottAgirs** [react-hook-form/react-hook-form#2642] [Q&A] [answered]
Redux Forms has this component called `FormSection />`, been looking through the docs to figure out if there's a similar concept in React-hook-forms or an example of how to imitate this?

[FormSection />](https://redux-form.com/8.3.0/docs/api/formsection.md/) - https://redux-form.com/8.3.0/docs/api/formsection.md/

From their docs: 

> The FormSection component makes it easy to split forms into smaller components that are reusable across multiple forms. It does this by prefixing the name of Field, Fields and FieldArray children, at any depth, with the value specified in the name prop.

**Comment by bluebill1049** [comment, marked as answer]
We don't.

**Reply by ScottAgirs** [reply]
Cool thanks for the reply, any chance you have a suggestion on how to tackle this?

**Reply by bluebill1049** [reply]
pass name group name as prop for your FormSection component or whatever you want to name it, `name={`${groupName}xxxxx`}`

**Reply by ScottAgirs** [reply]
Ahh ok, I had that idea too, hehe. Many thanks!
